e2e-tests/tests/upgrade/pxc/30-assert.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kuttl.dev/v1
+kind: TestAssert
+timeout: 100
+---
+apiVersion: pxc.percona.com/v1
+kind: PerconaXtraDBCluster
+metadata:
+  name: test-pxc-cluster
+spec:
+  crVersion: 1.12.0
```

Comment:
can we somehow get the `crVersion` here and in the other tests as the `PXC_OPERATOR_VERSION`/`PREVIOUS_PXC_OPERATOR_VERSION` env vars? otherwise we would still need to update all the tests when we update the values in the config which makes the whole idea of having them as variables less beneficial.

Reply:
Unfortunately no.

Reply:
Kuttl does not support it